internals/secrethub/run.go

```diff
@@ -460,9 +460,15 @@ func parseEnv(raw string) ([]envvar, error) {
 	vars := map[string]envvar{}
 	scanner := bufio.NewScanner(strings.NewReader(raw))
 
-	i := 1
+	i := 0
 	for scanner.Scan() {
+		i++
 		line := scanner.Text()
+
+		if line == "" || strings.HasPrefix(line, "#") {
```

Comment:
What happens when I have a line with only whitespace?

This could be a bit more foolproof solution:
```
if len(strings.TrimSpace(line)) == 0 || strings.HasPrefix(line, "#") {
   ...
}

Reply:
> What happens when I have a line with only whitespace?

Why would we want to allow this?

Reply:
Because in the user's experience it is the same as an empty line. I'd even go as far as saying that `     # comment` is still a comment. So then it'd become:
```
trimmed := string.TrimSpace(line)
if len(trimmed) == 0 || strings.HasPrefix(trimmed, "#") {
   ...
}
```